lib/ui/fixtures/ui_test.dart

```diff
@@ -233,6 +233,7 @@ void sendSemanticsUpdate() {
     childrenInHitTestOrder: childrenInHitTestOrder,
     additionalActions: additionalActions,
     headingLevel: 0,
+    linkUri: '',
```

Comment:
I think "URL" is more accurate here than "URI". The `<a>` tag and the `Link` widget only work with URIs that are also fetchable over the network via the same URI. Using arbitrary URIs as abstract identifiers without being able to fetch them is not something that's supported by this property, AFAICT. Unless there's something on the mobile side that extends beyond that?

Reply:
Done